pygmt/datasets/earth_free_air_anomaly.py

```diff
@@ -20,36 +20,43 @@ def load_earth_free_air_anomaly(
     ] = "01d",
     region: Sequence[float] | str | None = None,
     registration: Literal["gridline", "pixel", None] = None,
+    uncertainty: bool = False,
 ) -> xr.DataArray:
     r"""
-    Load the IGPP Earth free-air anomaly dataset in various resolutions.
+    Load the IGPP Earth free-air anomaly and uncertainty datasets in various
+    resolutions.
 
-    .. figure:: https://www.generic-mapping-tools.org/remote-datasets/_images/GMT_earth_faa.jpg
-       :width: 80 %
-       :align: center
+    .. list-table::
+       :widths: 50 50
+       :header-rows: 1
 
-       IGPP Earth free-air anomaly dataset.
+       * - IGPP Earth Free-Air Anomaly
+         - IGPP Earth Free-Air Anomaly Errors
```

Comment:
```suggestion
         - IGPP Earth Free-Air Anomaly Uncertainty
```

Reply:
Hm. I used the name for the upstream docs at https://www.generic-mapping-tools.org/remote-datasets/earth-faaerror.html, but was wondering, if uncertainty is better, as it is used in the docstrings.

Reply:
The GMT remote-dataset docs is inconsistenly using "error" and "uncertainty", while the officiail README file (https://topex.ucsd.edu/pub/global_grav_1min/README_V32.txt) uses "uncertainty":

> grav_error_32.1.nc  - uncertainty in gravity anomaly and sea surface slope, milligal or microradian, netcdf, lon-lat coordinates

Reply:
Made commit https://github.com/GenericMappingTools/remote-datasets/pull/128/commits/5aa5e5828a6ebbea5bc2a8b437d17e650e0277cb for changing this in the GMT remote-dataset docs.

But we stick to "error" in the GMT dataset name `faaerror` ?

Reply:
I think faaerror is fine.